text/0000-c-unwind-abi.md

```diff
@@ -0,0 +1,519 @@
+- Feature Name: `"C unwind" ABI`
+- Start Date: 2019-04-03
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Project group: [FFI-unwind][project-group]
+
+[project-group]: https://github.com/rust-lang/project-ffi-unwind
+
+# Summary
+[summary]: #summary
+
+We introduce a new ABI string, `"C unwind"`, to enable unwinding from other
+languages (such as C++) into Rust frames and from Rust into other languages.
+
+Additionally, we define the behavior for a limited number of
+previously-undefined cases when an unwind operation reaches a Rust function
+boundary with a non-`"Rust"`, non-`"C unwind"` ABI.
+
+As part of this specification, we introduce the term ["Plain Old Frame"
+(POF)][POF-definition]. These are frames that have no pending destructors and
+can be trivially deallocated.
+
+This RFC does not define the behavior of `catch_unwind` in a Rust frame being
```

Comment:
Can a non-POF frame be unwound by a foreign (non-forced) exception?

Reply:
Yes. POFs are only relevant to `longjmp` and similar constructs that "cancel" execution of a function.